Modules/Microsoft365DSC/Examples/Resources/IntuneAppCategory/1-Create.ps1

```diff
@@ -0,0 +1,28 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        Id                   = "f1fc9fe2-728d-4867-9a72-a61e18f8c606";
+        DisplayName          = "Books & Reference";
+    }
```

Comment:
`Ensure` property is missing (although it is implicitly set to `Present`). For clarity, we should always include it.